app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java

```diff
@@ -14,18 +14,24 @@
 import com.invisibleteam.goinvisible.R;
 import com.invisibleteam.goinvisible.databinding.ActivityEditBinding;
 import com.invisibleteam.goinvisible.model.ImageDetails;
+import com.invisibleteam.goinvisible.model.Tag;
 import com.invisibleteam.goinvisible.mvvm.common.CommonActivity;
 import com.invisibleteam.goinvisible.mvvm.edition.adapter.EditCompoundRecyclerView;
 import com.invisibleteam.goinvisible.mvvm.edition.dialog.EditDialog;
 
 import java.io.IOException;
+import java.util.List;
 
 import javax.annotation.Nullable;
 
 public class EditActivity extends CommonActivity {
 
     private static final String TAG_IMAGE_DETAILS = "extra_image_details";
     private static final String TAG = EditActivity.class.getSimpleName();
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    EditCompoundRecyclerView editCompoundRecyclerView;
```

Comment:
Yeah i agree, i forgot to change this but its not a matter of access but of design pattern. The setter is still there :)